crates/solver/src/liquidity.rs

```diff
@@ -212,8 +212,28 @@ impl LimitOrder {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct LimitOrderExecution {
+    /// The amount that the order `side` (`buy`, `sell`) should be filled by
+    /// this trade.
+    pub filled_amount: U256,
```

Comment:
Are these variable names matching exactly how the solvers would report this in their json response?

Reply:
No. The new value is currently being read from the `fee` field. Which already exists in our parsing logic but was not used AFAIK. If this is actually a problem we might have to introduce an entirely new name. Nothing is set in stone at this point so feel free to suggest something but if I recall correctly you also called the field `fee` in your overview document.

Reply:
Actually [here](https://github.com/cowprotocol/services/blob/main/crates/shared/src/http_solver/model.rs#L261-L269) are the fields we currently read from the solver response. So turns out that I actually misremembered slightly as the existing (but currently unused) format is actually `fee: {amount: "0", token: "0xeee..eeee"}`. But I would rather turn that into simple `U256` which implicitly refers to orders `sell_token`. WDYT?

Reply:
I see. Yeah `fee` is fine so let's stick with it. Regarding the format, I guess for uniformity reasons (since we send it to solvers in the fee -> {amount, token} format in the input.json), we can keep it as is.

Reply:
And regarding specifying the executed amounts, depending on whether it's a sell or buy order, the solver would submit an `exec_sell_amount` or `exec_buy_amount`, accordingly, right?

Reply:
Right now the parsing logic always expects both values but we only use one and and recompute the other based on the ucp.